src/pages/components/anchor.mdx

```diff
@@ -73,6 +73,8 @@ import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExterna
 
 <Component components={props.data.mdx.components} componentName="anchor" />
 
+The Anchor component applies appropriate attributes, icon and styles based on the props provided.
```

Comment:
Oxford comma before "and" per Zendesk content guidelines! https://garden.zendesk.com/content/punctuation#comma

Reply:
Updated! I always forget that rule. 🙈